packages/v4/src/content/design-guidelines/components/file-upload/file-upload.md

```diff
@@ -2,4 +2,46 @@
 id: File upload
 section: components
 ---
-A **file upload** component uploads the contents of a file into the browser for further processing. Once uploaded, content may optionally be edited. While the component can upload any file, it is primarily designed for use with text files where the context can easily be displayed in the text area.
+A **file upload** component allows the users to upload a single file into the browser. 
+
+## Usage
+
+Use a single file upload component when only one file can be uploaded. If multiple files need to be uploaded, use the [multiple file upload component](https://www.patternfly.org/v4/components/file-upload---multiple/). This component is recommended for the use of text files where content can easily be displayed in the text area, but all file types are supported.
+
+Once the file is uploaded, the user can edit the content.
+
+To upload, a user can either:
+- Drag and drop the file into the box. When the user hovers over the file upload component, a message should appear telling the user they can drop files there. 
+- Browse their device for a file to upload.
+
+
+## Elements
+<img src="./img/File upload.png" alt="File Upload"> 
```

Comment:
- The name of the file is `Fileupload.png` (without the space) so this is throwing an error that it can't find this file (with the space).
- Needs a `/` before the closing angle bracket `>`
```suggestion
<img src="./img/Fileupload.png" alt="File Upload" /> 
```